parsing/parse_hrp.py

```diff
@@ -256,7 +273,13 @@ def main():
         print(f"Error: File '{perf_log_path}' does not exist.")
         sys.exit(1)
 
-    parse_hrperf_log(perf_log_path, max_frequency_ghz)
+    try:
```

Comment:
Let's take the chance to refactor the code to use `argparse` ([doc](https://docs.python.org/3/library/argparse.html)) to handle the argument parsing. 

Ideally, I'd imagine three new flags:
1. `--raw_counter`
1. `--tsc_ts`
1. `--tsc_freq <TSC_FREQUENCY>`

The first two flags are essentially boolean. Default action to `true` if they are specified and `false` if they are not provided.

If `raw_counter` is provided, we output additional raw counter numbers, without dividing the time duration.
If `tsc_ts` is provided, we divide the timestamp by the `tsc_freq`. If not, use timestamps directly as we used to.

We can make `--tsc_freq` as a required argument, so that it can replace the old `max_frequency_ghz` argument.